packages/react-core/src/utils/transformTileCoordsToWGS84.js

```diff
@@ -0,0 +1,66 @@
+import { lerp } from '@math.gl/core';
+import { lngLatToWorld, worldToLngLat } from '@math.gl/web-mercator';
+
+/**
+ * Transform WGS84 coordinates to tile coords.
+ * It's the inverse of deck.gl coordinate-transform (https://github.com/visgl/deck.gl/blob/master/modules/geo-layers/src/mvt-layer/coordinate-transform.js)
```

Comment:
This link leads to a 404, or am I doing something wrong?

Reply:
Fixed